tests/unit/admin/views/test_projects.py

```diff
@@ -342,3 +343,26 @@ def test_non_normalized_name(self, db_request):
         )
         with pytest.raises(HTTPMovedPermanently):
             views.journals_list(project, db_request)
+
+
+class TestProjectSetLimit:
+    def test_sets_integer_limit(self, db_request):
+        project = ProjectFactory.create()
+
+        db_request.user = UserFactory.create()
+        db_request.current_route_path = lambda: "/admin/projects/",
```

Comment:
You'll want to add a similar thing for ``db_request.route_path()`` to fix the testing error. I normally use a ``pretend.call_recorder()`` so I can make sure that it is called with the correct arguments.

Reply:
Ah I get now. Thanks. Done and added two additional tests for coverage.